src/middleware/SecureTrafficTenantLevel.ts

```diff
@@ -0,0 +1,21 @@
+import * as builder from "botbuilder";
+const config = require("config");
+// Strip bot mentions from the message text
+export class SecureTrafficTenantLevel implements builder.IMiddlewareMap {
+
+    public readonly botbuilder = (session: builder.Session, next: Function): void => {
+        let targetTenant = typeof(config.OFFICE_365_TENANT_FILTER) !== "undefined" ? config.OFFICE_365_TENANT_FILTER : null;
+        let currentMsgTenant = typeof(session.message.sourceEvent.tenant) !== "undefined" ? session.message.sourceEvent.tenant.id : null;
+        if (targetTenant !== null && targetTenant !== "#ANY#") {
```

Comment:
What's the point of "#ANY#" anyway? Can we just have no filter == let all messages through?

Reply:
I have already implemented the same way in C# Sample App so was following the same pattern in NodeJs as well, no value for target tenant is also a good idea to let all the message through, I can update both the samples with no value in target tenant, let me know if you want me to keep as it is or want to remove  #ANY#.